pkg/drivers/kic/oci/oci.go

```diff
@@ -190,7 +190,11 @@ func CreateContainerNode(p CreateParams) error { //nolint to suppress cyclomatic
 		runArgs = append(runArgs, "--network", p.Network)
 		runArgs = append(runArgs, "--ip", p.IP)
 	}
-	if p.GPUs != "" {
+
+	if p.GPUs == "all" || p.GPUs == "amd" {
+		runArgs = append(runArgs, "--device", "/dev/kfd", "--device", "/dev/dri", "--group-add", "video", "--group-add", "render")
```

Comment:
Also, if you don't have an AMD card will this fail? We may want to deprecate the `--gpus=all` flag.

Reply:
Behaviour wont change compared to the Nvidia behaviour.  If you try without nvidia if will fail.

Reply:
Actually, I can only enable if specifically specified as 'amd'.

Reply:
Yeah, the concern is we may have Nvidia users who have scripts or whatnot set to use `--gpus=all` but then the next release will also add these flags, and I'm worried we may cause a regression.